### ISSUE TITLE
Does it work with spring boot 3 and java 17?

### ISSUE THREAD

**Issue by m-valeev** [patka/cassandra-migration#75]
I use this library with spring boot 3 and java 17.
Configuration class CassandraMigrationAutoConfiguration is not initialised by spring context. Looks like it is just ignored. I have no any exception when application starts and migrations just not runs.
I have used it with spring boot 2.7.5 and java 17 and it worked.

**Comment by m-valeev** [comment]
Have found that spring boot 3 ignores spring.factories files. So auto-configuration CassandraMigrationAutoConfiguration doesn't work

**Comment by patka** [comment]
Hi, thanks for the research. I never tested it with Spring Boot 3 but that for sure is something to look into before it becomes a major issue for more people. Thanks for highlighting it, I see that I reserve some time for it in the next couple of weeks. I cannot give an ETA as currently sick kids always cross my plans.

**Comment by qvistgaard** [comment]
We are using this library and we are in the process of switching to spring boot 3. For our applications. I can report adding the file, with below content make is load:

`src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports`
```
org.cognitor.cassandra.migration.spring.CassandraMigrationAutoConfiguration
```
~~There is another caveat though. Spring Data Cassandra changed property names, and they are no longer `spring.data.cassandra.` but instead `spring.cassandra.`. I worked around that issue by mapping it using spel like this:~~
```
spring.data.cassandra.keyspace-name=${spring.cassandra.keyspace-name}
```
I realized above is my own doing ;)

It might be an easy fix without to many complications. I will have a PR ready in a few minutes
